client/state/data-layer/wpcom/i18n/locale-suggestions/test/index.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { fetchLocaleSuggestions, localeSuggestionsReceive } from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { receiveLocaleSuggestions } from 'state/i18n/locale-suggestions/actions';
+
+describe( 'locale suggestions request', () => {
+	describe( 'successful requests', () => {
+		test( 'should return HTTP request to /locale-guess endpoint', () => {
+			const action = { type: 'SOME_ACTION' };
+
+			expect( fetchLocaleSuggestions( action ) ).toEqual(
+				http(
+					{
+						apiVersion: '1.1',
+						method: 'GET',
+						path: '/locale-guess',
+					},
+					action
+				)
+			);
+		} );
```

Comment:
just to post my opinion, I don't think these tests are helping us. they are testing things that are tested elsewhere. @ramonjd you can ignore me here because others dissent on this point.

Reply:
I don't have strong feelings here, but the tests can help when we have more complex success/error handlers where we dispatch a number of actions to update separate parts of the state tree

Reply:
I'm also non-committal in this case. 

I'm no purist, but am usually defensive when it comes to tests, and not just due to @gwwar's observations. I like to ensure that any alteration in expected behaviour, even if we're dealing with a return value, is a deliberate one. 

The above notwithstanding, though we might be updating the endpoint to return more useful suggestions soon, I don't foresee any extra complexity for the state tree or this module. Given that, and in light of the existing middleware tests, I'll take the test suite out for now.

Reply:
> I'm no purist, but am usually defensive when it comes to tests, and not just due to @gwwar's observations. I like to ensure that any alteration in expected behaviour, even if we're dealing with a return value, is a deliberate one.

Yeah this is where I like to encourage us to use tools meant for these purposes. It's not a test, it's just a notification that something changed, and usually if _anything_ changes it will be legitimate.

[code owners](https://github.com/blog/2392-introducing-code-owners) is one such feature